assets/cubyz/blocks/sand.json

```diff
@@ -1,11 +1,9 @@
-{
-	"class" : "sand",
-	"hardness" : 5,
-	"drops" : [
-		"auto"
-	],
-	"model" : "cube",
-	"texture_top" : "cubyz:sand",
-	"texture" : "cubyz:sand_side",
-	"texture_bottom" : "cubyz:sand"
-}
+{
+	"class" : "sand",
+	"hardness" : 5,
+	"drops" : [
+		"auto"
+	],
+	"model" : "cube",
+	"texture" : "cubyz:sand",
+}

```

Comment:
It seems like you accidentally messed up with the line endings again, which is why github is showing so many line changes here. Please try to fix your setup so this doesn't happen again.

Reply:
thanks for fixing it, sorry! i misread your message about this in my first PR and set all my line endings to CRLF..... i fixed that in my editor, shouldn't happen again!